jose-jwt/jwe/JWE.cs

```diff
@@ -0,0 +1,330 @@
+﻿#if NETSTANDARD
+namespace Jose.jwe

```

Comment:
Probably will be nicer to rename to: `Jose.Jwe`

Reply:
Now done.
Also note class Jwe renamed to class JWE to be consistent with class naming for JWT, and to avoid name clash with the namespace.

Reply:
THanks !